src/api/app/views/webui/projects/maintained_projects/index.html.haml

```diff
@@ -6,7 +6,12 @@
 .card
   = render partial: 'webui/project/tabs', locals: { project: @project }
   .card-body
-    %h3 Maintained Projects
+    %h3
+      Maintained Projects
+      - if feature_enabled?(:responsive_ux)
```

Comment:
We should also check for `update_policy` here, no?

So it would be:
```suggestion
      - if update_policy && feature_enabled?(:responsive_ux)
```

Reply:
Oh yes, I forgot